redash/query_runner/presto.py

```diff
@@ -116,7 +116,10 @@ def run_query(self, query, user):
         except DatabaseError as db:
             json_data = None
             default_message = 'Unspecified DatabaseError: {0}'.format(db.message)
-            message = db.message.get('failureInfo', {'message', None}).get('message')
+            if isattr(db.message, 'get'):
```

Comment:
Are you sure `isattr` is a Python function?

As an alternative, you can just check if it's a `dict` (`if isinstance(db.message, dict)`).

Reply:
Yes. My mistake. Should be hasattr function. I must accidentally made the typo when changed the if from the one-line form to the multi-line form (due to the line-too-long stylecheck error).